docs/en/siem/machine-learning.asciidoc

```diff
@@ -28,10 +28,11 @@ the job's `Group` field ({kib} -> {ml-cap} -> Create/Edit job -> Job details).
 
 [float]
 [[included-jobs]]
-==== Prebuilt Jobs
+==== Prebuilt jobs
 
-The SIEM app ships with prebuilt Machine Learning Jobs for detecting anomalies.
-If you ship data using https://www.elastic.co/products/beats[Beats] and your environment
+The SIEM app ships with prebuilt {ml} {anomaly-jobs} for
+automatically detecting host and network anomalies. If you ship data using
+https://www.elastic.co/products/beats[Beats] and your environment
 is configured with the appropriate indices (`auditbeat-*`, `filebeat-*`, `packetbeat-*`,
 or `winlogbeat-*`) via {kib} -> Management -> Index Patterns, the jobs will be installed
 on page load, and will be displayed within the `Anomaly Detection` interface.
```

Comment:
For 7.5, as of https://github.com/elastic/kibana/pull/48067 this is no longer the case. All jobs will now be displayed by default/regardless of indices configured. Installation still happens in the background, but only when a user _enables_ a given job do we attempt to install the corresponding ML module. I don't think this aspect (installing) needs highlighted as it should make no difference to the user whether the job is _installed_ or _not_, just whether it is running or not.